packages/pigeon/platform_tests/test_plugin/ios/Classes/CoreTests.gen.swift

```diff
@@ -50,20 +50,20 @@ struct AllTypes {
   var a4ByteArray: FlutterStandardTypedData
   var a8ByteArray: FlutterStandardTypedData
   var aFloatArray: FlutterStandardTypedData
-  var aList: [Any?]
+  var aList: [Any]
   var aMap: [AnyHashable: Any?]
   var anEnum: AnEnum
   var aString: String
 
-  static func fromList(_ list: [Any?]) -> AllTypes? {
+  static func fromList(_ list: [Any]) -> AllTypes? {
     let aBool = list[0] as! Bool
     let anInt = list[1] as! Int32
     let aDouble = list[2] as! Double
     let aByteArray = list[3] as! FlutterStandardTypedData
     let a4ByteArray = list[4] as! FlutterStandardTypedData
     let a8ByteArray = list[5] as! FlutterStandardTypedData
     let aFloatArray = list[6] as! FlutterStandardTypedData
-    let aList = list[7] as! [Any?]
+    let aList = list[7] as! [Any]
```

Comment:
do we want to test out both `[Any?]` and `[Any]`

Reply:
What do you mean? I'm not sure why we would need Any? for anything in this context since Any can already be nil

Reply:
Oh interesting - what about the maps below? Do you also want to remove the `?`

e.g.   

```
var aNullableMap: [AnyHashable: Any?]? = nil
var nullableMapWithObject: [String?: Any?]? = nil
```

Reply:
probably a good idea

Reply:
> I'm not sure why we would need Any? for anything in this context since Any can already be nil

Actually, `Any` cannot represent `nil` in Swift: 

![Screenshot 2023-02-23 at 4 13 41 PM](https://user-images.githubusercontent.com/41930132/221060597-6b165e49-467e-473f-9fd7-93295dc6ce14.png)

Reply:
Hmmm, if I provide concrete type of nil then it works. I think it's fine to keep `Any` then. Though I am not sure how this is related to the `Bool?` crash you had the other day. May be helpful if you can provide a code snippet so I can try out in playground.

Reply:
The `Bool?` crash was `nil` attempting to be cast to `Bool?`, it failed for the reasons mentioned here. I'm not sure how to replicate it without a lot of extra steps.